core/drivers/pl022_spi.c

```diff
@@ -356,6 +271,31 @@ static inline uint32_t pl022_calc_freq(struct pl022_data *pd,
 	return pd->clk_hz / (cpsdvr * (1 + scr));
 }
 
+static void pl022_control_cs(struct spi_chip *chip, enum gpio_level value)
+{
+	bool is_busy = false;
+	struct pl022_data *pd = container_of(chip, struct pl022_data, chip);
+
```

Comment:
Yes, it's valid in SPI for users to not give GPIO data for CS and control CS GPIO pin manually. This allows them to use any GPIO pin they wish for CS.